django/contrib/postgres/apps.py

```diff
@@ -18,6 +18,11 @@ def ready(self):
             if conn.vendor == 'postgresql':
                 conn.introspection.data_types_reverse.update({
                     3802: 'django.contrib.postgresql.fields.JSONField',
+                    3904: 'django.contrib.postgresql.fields.IntegerRangeField',
+                    3906: 'django.contrib.postgresql.fields.FloatRangeField',
+                    3910: 'django.contrib.postgresql.fields.DateTimeRangeField',
+                    3912: 'django.contrib.postgresql.fields.DateRangeField',
+                    3926: 'django.contrib.postgresql.fields.BigIntegerRangeField',
                 })
```

Comment:
I've realized something about this - if, for whatever reason (perhaps custom subclassing??), multiple apps are installed that can introspect these types, if this pattern is copied it's the *last* app in `INSTALLED_APPS` that take precedence, even though the general rule with Django is apps earlier in `INSTALLED_APPS` take precedence. Idk if it's a huge issue though.

Reply:
🤐